nosnoc/model.py

```diff
@@ -57,23 +57,23 @@ def __init__(self,
                  c: Optional[List[ca.SX]] = None,
                  S: Optional[List[np.ndarray]] = None,
                  g_Stewart: Optional[List[ca.SX]] = None,
-                 u: ca.SX = ca.SX.sym('u_dummy', 0, 1),
-                 z: ca.SX = ca.SX.sym('z_dummy', 0, 1),
+                 u: Optional[ca.SX] = ca.SX.sym('u_dummy', 0, 1),
```

Comment:
These should not be `Optional`. `Optional` is explicitly the union between the type argument and `NoneType`. These should never be `None`.